ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -174,16 +174,18 @@ protected override async Task ValidateAndSave()
                 ModelState.AddModelError("ClinicalDetails.IsMDRTreatment", ValidationMessages.MDRCantChange);
             }
 
-            if ((ClinicalDetails.IsPostMortem ?? false))
+            if (Notification.NotificationStatus == NotificationStatus.Notified
```

Comment:
As discussed this morning, we want to get rid of this condition

Reply:
I haven't completely removed this condition as actually it would allow behaviour that could accidentally avoid the validation.
Eg. if you were creating a draft
- selected `PostMortem = true`
- saved and moved to a different page
- moved back to the `ClinicalDetails` page
You could now set `StartedTreatment = true`, just because post mortem has been set in a previous save. But actually as the notification hasn't been submitted yet, it makes more sense to always do this check for drafts. 
I know this goes against the ACs slightly, but if you agree I will change the ACs on the ticket.

Reply:
The counter-argument to this would be that you can already do this for notified notifications so does it make any sense to handle it differently to drafts? I'm not sure.

Reply:
I think for now we should go with what's in the ACs - we can then potentially discuss again when the ticket gets QAed

Reply:
The answer may well be that we just don't care enough about this issue to invest more time in it